### PR TITLE
E_CLASSROOM-293 [Admin] [FE/BE] Categories search by name

### PR DESCRIPTION
**Pull request by ErickDoroy**
### Issue Link
- https://framgiaph.backlog.com/view/E_CLASSROOM-293
### Definition of Done
- [x] Follow design in Figma
- [x] Should be able to search Category by name

## Related PR: 
- https://github.com/framgia/sph-classroom-els-be/pull/74
#### Main note
- http://localhost:3003/admin/categories?page=1&search=&sortBy=&sortDirection=
#### Pages Affected
- N/A

### Scenarios/ Test Cases
- [ ] Able to search category by name 
#### User Interface
- N/A
#### User Experience 
- N/A
#### Functionality
- N/A
### Screenshot
categorysearch

### CODE REVIEW

**Comment by catampongan** [comment]
## Working
- [x] Able to search category by name
## Comment
- [x] Instead of "search name or email", make it "search category name"
------------------
Task-293
Task-293-1
Task-293-bug

---------------------
- Total Test Time: 0.25 hr

**Comment by catampongan** [comment]
## Approved PR #142 
- [x] Design is the same in Figma
- [x] Able to search by category name
-----------------
Task-293

Task-293-1

---------------
- Total Test Time: 0.25 hr
- Total Iterations: 1
- Total Comments: 1

File: src/pages/admin/categories/CategoryList/index.js [hunk 1]
```diff
@@ -99,12 +116,15 @@ const CategoryList = () => {
...
+            <Form className={style.searchSection} onSubmit={onSearchSubmit}>
               <div className={style.searchInput}>
                 <Form.Control
                   className={style.searchBar}
                   type="text"
-                  placeholder="Search name or email"
+                  aria-label="Search"
+                  value={search}
+                  onChange={onChangeData}
+                  placeholder="Search category name"
```
Comment (karlolee50) [inline src/pages/admin/categories/CategoryList/index.js]: please use the SearchBar component

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 5 minutes
Comment/s: 1

**Review by karlolee50 — APPROVED** [review]

**Pull request merged**